packages/circle-react-elements/src/components/TransactionDetails/TransactionDetails.tsx

```diff
@@ -0,0 +1,79 @@
+import { ReactNode, useMemo } from 'react';
+
+import { TransactionType } from '../../lib/constants';
+import { formatDate, shortenHash } from '../../lib/format';
+import { TransactionWithToken } from '../../lib/types';
+import { ChainLabel } from '../ChainLabel';
+import { useGetTransaction } from '../hooks/useGetTransaction';
+import { TokenItem } from '../TokenItem';
+import { TransactionStateText } from '../TransactionStatusText';
+
+export interface TransactionDetailsProps {
```

Comment:
These props are actually a bit confusing. If, as a developer, I’m already providing TransactionWithToken as props, why do I also need to provide a getTransaction function? I understand it is used internally for refetching, but this should be better explained in the documentation.

Reply:
removed it